rules/internal/testutils/parse.go

```diff
@@ -80,3 +102,69 @@ func ParseProto3Tmpl(t *testing.T, src string, data interface{}) *desc.FileDescr
 	// Parse the proto as a string.
 	return ParseProto3String(t, protoBytes.String())
 }
+
+// decompile reads compiled Go protos and returns a map of strings
+// corresponding to the original protos.
+//
+// This is used to handle google.api common protos, which some tests need
+// to use.
+func decompile(filenames ...string) map[string]string {
+	var fdps []*dpb.FileDescriptorProto
+	for _, filename := range filenames {
+		// Compiled golang protos include a compressed descriptor, which must be
+		// uncompressed before it can be populated into a FileDescriptorProto.
+		compressedDesc := proto.FileDescriptor(filename)
+		r, err := gzip.NewReader(bytes.NewReader(compressedDesc))
+		if err != nil {
+			panic(fmt.Errorf("Corrupted compressed file %q: %v", filename, err))
```

Comment:
Since this function is used for testing, may be using `testing.T` to `Fatalf` is better.

Reply:
It occurs at module setup. We do not want to do this (somewhat expensive) work on every test.